### PR TITLE
update faq.md (fixes #607)

### LINKED ISSUE #607: Adding progress page link in the relevant FAQ
### Problem
The FAQ Q8 answers how many PRs and Issues are needed to be completed but does not provide a way to check the progress. The only way someone would know about the progress page is if they read carefully the first steps page and find it mentioned there only once. How about we add the link to the progress page in the answer to this question as well?

 ### Steps to reproduce the problem
visit Q8 in https://treehouses.io/#!./pages/vi/faq.md#Helpful_Links

 ### Screenshots
Capture

 ### Proposed solution
The link to the progress page can be added to the answer:
https://treehouses.io/#!./pages/vi/trackprogress.md

### PR DESCRIPTION
**Pull request by ali18997**
Fixes #607

### Description
Added progress tracker link to Q8.

### Raw.Githack preview link
https://raw.githack.com/ali18997/ali18997.github.io/update1/#!pages/vi/faq.md

### CODE REVIEW

**Review by law-lin — APPROVED** [review]
You should include the raw.githack link too when you make the PR.
Here's the link:
https://raw.githack.com/ali18997/ali18997.github.io/update1/#!pages/vi/faq.md

**Review by LordJashin32 — APPROVED** [review]
Thanks @law-lin for the link. This looks like a good addition to the FAQ. Cheers.

File: pages/vi/faq.md [hunk 1]
```diff
@@ -53,7 +53,7 @@
 
 #### Q8: How many pull requests and issues in total do I need to make to finish the steps?
 
-+ To finish the initial steps, you need to make a **minimum** of 4 issues and 5 pull requests so you can get familiar with GitHub and to show us that you can write proper issues/PRs.
++ To finish the initial steps, you need to make a **minimum** of 4 issues and 5 pull requests so you can get familiar with GitHub and to show us that you can write proper issues/PRs. Check your progress [here](https://treehouses.io/#!./pages/vi/trackprogress.md).
```
Comment (dogi) [inline pages/vi/faq.md]: @ali18997 please do not use and absolute URL (no https necessary)

Comment (ali18997) [reply pages/vi/faq.md]: Got it, made the change

**Review by dogi — APPROVED** [review]

**Pull request merged**